src/azure-cli/azure/cli/command_modules/rdbms/tests/latest/test_rdbms_commands.py

```diff
@@ -89,6 +90,62 @@ def test_mysql_server_mgmt(self, resource_group_1, resource_group_2):
     def test_postgres_server_mgmt(self, resource_group_1, resource_group_2):
         self._test_server_mgmt('postgres', resource_group_1, resource_group_2)
 
+    @AllowLargeResponse()
+    @ResourceGroupPreparer(parameter_name='resource_group_1')
+    @live_only()
+    def test_mariadb_server_mgmt_public_paramter(self, resource_group_1):
+        self._test_server_mgmt_public_paramter('mariadb', resource_group_1)
+
+    @AllowLargeResponse()
+    @ResourceGroupPreparer(parameter_name='resource_group_1')
+    @live_only()
+    def test_mysql_server_mgmt_public_paramter(self, resource_group_1):
+        self._test_server_mgmt_public_paramter('mysql', resource_group_1)
+
+    @AllowLargeResponse()
+    @ResourceGroupPreparer(parameter_name='resource_group_1')
+    @live_only()
```

Comment:
May I know the reason to make it live only here?

Reply:
This is because the firewall name consists of the timestamp( to keep it unique). Recording the test with non-deterministic value like the timestamp would fail the test always.